README.md

```diff
@@ -60,7 +60,7 @@ $ npm install -g @salesforce/commerce
 $ sfdx COMMAND
 running command...
 $ sfdx (-v|--version|version)
-@salesforce/commerce/250.0.0 darwin-x64 node-v18.16.1
+@salesforce/commerce/250.1.0 darwin-x64 node-v18.16.1
```

Comment:
Any reason to bump this version in read me?

Reply:
It was supposed to bump earlier during release, `yarn version` didn't run in the pipeline